website/tests/netlify/bm25.test.js

```
@@ -0,0 +1,284 @@
+// We need to extract the internal functions for testing
+// Read the search.js file and evaluate the functions
+const fs = require('fs');
+const path = require('path');
+
+// Load search.js and extract functions
+const searchPath = path.resolve(__dirname, '../../netlify/functions/search.js');
+const searchCode = fs.readFileSync(searchPath, 'utf-8');
+
+// Extract and evaluate the tokenize function
+const tokenizeMatch = searchCode.match(/function tokenize\(text\) \{[\s\S]*?\n\}/);
+const tokenize = eval('(' + tokenizeMatch[0] + ')');
```

Biome (2.3.13)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)